src/main/java/foodwhere/storage/JsonSerializableAddressBook.java

```diff
@@ -26,8 +26,7 @@ class JsonSerializableAddressBook {
     /**
      * Constructs a {@code JsonSerializableAddressBook} with the given stalls.
      */
-    @JsonCreator
-    public JsonSerializableAddressBook(@JsonProperty("stalls") List<JsonAdaptedStall> stalls) {
+    @JsonCreator public JsonSerializableAddressBook(@JsonProperty("stalls") List<JsonAdaptedStall> stalls) {
```

Comment:
Is it better to leave the @JsonCreator on a newline instead of putting them together?

Reply:
Ah, didn't merge the code properly. Reverting this change.